src/Microsoft.AspNet.Server.Kestrel/Http/FrameRequestStream.cs

```diff
@@ -102,5 +109,11 @@ public override void Write(byte[] buffer, int offset, int count)
         {
             throw new NotImplementedException();
         }
+
+        protected override void Dispose(bool disposing)
```

Comment:
Use an internal API instead, you don't want random application code to dispose the streams.